lib/environment.ts

```diff
@@ -1,8 +1,8 @@
 import fs from "fs";
-import fetch from "node-fetch";
+import "isomorphic-unfetch";
```

Comment:
You can import Next.js to include their polyfill for fetch in Node in tests (if you run the app it's done automatically)

In Jest's `setupTestAfterEnv` file add this:

```ts
import "next";
```

Example: https://github.com/saleor/react-storefront/blob/fix-fixtures-import/apps/saleor-app-checkout/setupTestsAfterEnv.js